test/test_integration_role.py

```diff
@@ -129,6 +129,84 @@ def test_role_exists_policy_returns_result(self):
+    def test_role_memberships_short_with_help_returns_role_memberships_help(self):
+        output = self.invoke_cli(self.cli_auth_params,
+                                 ['role', 'memberships', '-h'])
+        self.assertIn("Name:\n  memberships", output)
+
+    @integration_test(True)
+    def test_role_memberships_long_with_help_returns_role_memberships_help(self):
+        output = self.invoke_cli(self.cli_auth_params,
+                                 ['role', 'memberships', '--help'])
+        self.assertIn("Name:\n  memberships", output)
+
+    @integration_test()
+    def test_role_memberships_without_id_returns_help(self):
+        with redirect_stderr(self.capture_stream):
+            self.invoke_cli(self.cli_auth_params,
+                            ['role', 'memberships'], exit_code=1)
+        self.assertIn("Error the following arguments are required:", self.capture_stream.getvalue())
+
+    @integration_test()
+    def test_role_memberships_without_prefix_returns_help(self):
+        with redirect_stderr(self.capture_stream):
+            self.invoke_cli(self.cli_auth_params,
+                            ['role', 'memberships', '-i', 'someuser'], exit_code=1)
+        self.assertIn("Error the following arguments are required:", self.capture_stream.getvalue())
+
+    @integration_test(True)
+    def test_role_memberships_long_role_id_returns_result(self):
+        output = self.invoke_cli(self.cli_auth_params,
+                                 ['role', 'memberships', '--id=user:someuser'])
+        self.assertIn('user:someuser', output)
+
+    @integration_test(True)
+    def test_role_memberships_short_options_returns_result(self):
+        output = self.invoke_cli(self.cli_auth_params,
+                                 ['role', 'memberships', '-i', 'user:someuser'])
+        self.assertIn('user:someuser', output)
+
+    @integration_test(True)
+    def test_role_memberships_host_returns_result(self):
+        output = self.invoke_cli(self.cli_auth_params,
+                                 ['role', 'memberships', '-i', 'host:anotherhost'])
+        self.assertIn('host:anotherhost', output)
+
+    @integration_test(True)
+    def test_role_memberships_layer_returns_result(self):
+        output = self.invoke_cli(self.cli_auth_params,
+                                 ['role', 'memberships', '-i', 'layer:somelayer'])
+        self.assertIn('layer:somelayer', output)
+
+    @integration_test(True)
+    def test_role_memberships_group_returns_result(self):
+        output = self.invoke_cli(self.cli_auth_params,
+                                 ['role', 'memberships', '-i', 'group:somegroup'])
+        self.assertIn('group:somegroup', output)
+
+    @integration_test(True)
+    def test_role_memberships_policy_returns_result(self):
+        output = self.invoke_cli(self.cli_auth_params,
+                                 ['role', 'memberships', '-i', 'policy:root'])
+        self.assertIn('policy:root', output)
+
```

Comment:
Were you planning on adding more integration tests for things like direct vs. indirect memberships? I think that would be valuable given that the API repo doesn't have in depth integration tests.

Reply:
Good thought - pushed up a few more tests to validate the memberships returned.

Reply:
Great. I approved https://github.com/cyberark/conjur-api-python/pull/33 and will finish reviewing this one now.